manim/mobject/coordinate_systems.py

```diff
@@ -1423,7 +1423,10 @@ class NumberPlane(Axes):
     kwargs : Any
         Additional arguments to be passed to :class:`Axes`.
 
-    .. note:: If :attr:`x_length` or :attr:`y_length` are not defined, the plane automatically adjusts its lengths based
+
+    .. note::
+
+        If :attr:`x_length` or :attr:`y_length` are not defined, the plane automatically adjusts its lengths based
         on the :attr:`x_range` and :attr:`y_range` values to set the unit_size to 1.
```

Comment:
Just a minor formatting change, while we are at it.

```suggestion
        on the :attr:`x_range` and :attr:`y_range` values to set the ``unit_size`` to 1.
```